node/src/dev_service.rs

```diff
@@ -232,6 +234,9 @@ pub fn new_full(
 		})
 	};
 
+	let telemetry_span = TelemetrySpan::new();
+	let _telemetry_span_entered = telemetry_span.enter();
```

Comment:
Where did these come from? I wonder if this is why relay chain telemetry didn't seem to work in moonbeam before.

Reply:
https://github.com/paritytech/cumulus/commit/28d1d00747e1e60a3e26f0112b4580c805fa1557